bitcoinplusplus/layer-2/exploring-p2p-hashrate-markets-v2.md

```diff
@@ -218,44 +236,52 @@ And cloud mining, you're getting price gouged almost always.
 I've never seen a cloud mining contract where you make any money.
 On the contrary, you're usually losing quite a bit.
 And hosting is really, like I'm saying, not only do you have this asymmetric trust where each person here can rug you, but you also just have premiums, right, that get put in at every level there.
-So a good way we can try and avoid this is just go straight between the miner and the buyer, right?
+So a good way we can try and avoid this is just go straight between the miner and the buyer.
 Just kind of try and create a market where you have the two that are talking to each other directly.
 Before getting there.
 So we've had some hash rate futures markets.
 FTX had launched one last year.
 Luxor has something which is a little bit more legit, which is a non-deliverable future.
 So it's a swap, basically.
 It's a lot more traditional.
-The problem with these is that we haven't seen much liquidity, right?
+The problem with these is that we haven't seen much liquidity.
 And again, this kind of goes back to, I think, what I had mentioned before, is that why are we going to these traditional financial institutions looking for money when they're the ones that like, usually, one, they don't understand Bitcoin, they have no real incentive to understand or try and take that risk on Bitcoin outside of the fact that they're gonna try and be relatively predatory on the fact that, oh, you need the money and no one else is gonna give you the money, so I'm gonna give it to you for like really expensively.
 And yeah, it creates this situation, right, where nobody's really out trying to buy these future hash rate contracts in the traditional financial markets.
 Other considerations are, well, what if you want to actually take delivery?
 There's nothing really like that.
 What about the duration of this contract?
-So for the most part, if you look at marketplaces like NiceHash and Mining Regret tools, you have pretty short contract durations there, which makes it okay if you're kind of trying to do the betting in one direction or another, but it's fairly short, right?
+So for the most part, if you look at marketplaces like NiceHash and Mining Regret tools, you have pretty short contract durations there, which makes it okay if you're kind of trying to do the betting in one direction or another, but it's fairly short.
 So you're not really buying something and then just kind of letting it sit there and do its thing.
 And then in terms of the actual non-deliverables or like the swaps, like contracts, well, it's, you know, you're buying kind of like a traditional financial instrument and it's not code.
 Oh, wow, okay.
 That was fast.
 Well, I'm gonna run through this really quick.
 Thank you.
```

Comment:
It seems these lines are the speaker's reaction to being informed that their time is almost up. Without additional context, this moment might not be fully understood by someone reading the transcript. Providing a brief explanatory note could help clarify this situation for readers, ensuring they grasp the context of the speaker's sudden shift in pace. For example:
```suggestion
[Note: Speaker reacts to being informed they are almost out of time.]: "Oh, wow, okay.
That was fast.
Well, I'm gonna run through this really quick.
Thank you."
```